integration_tests/run_pyspark_from_build.sh

```diff
@@ -115,15 +115,22 @@ else
           $RUN_TEST_PARAMS \
           "$@"
     else
+        ## 'REMOTE_RES_PATH' is set in the cloud environment to support distributed file path
+        if [ -n "$REMOTE_RES_PATH" ]; then
+            PYTEST_RES_PARAMS="$REMOTE_RES_PATH/../runtests.py \
+              --std_input_path=$REMOTE_RES_PATH/src/test/resources/"
+        else
+            PYTEST_RES_PARAMS="$SCRIPTPATH/runtests.py --rootdir $SCRIPTPATH $SCRIPTPATH/src/main/python \
```

Comment:
seems `--rootdir` option does not support hdfs:/xxx or s3a://xxx PATH, only support file path?

 --rootdir=ROOTDIR     Define root directory for tests. Can be relative path: 'root_dir', './root_dir',
                        'root_dir/another_dir/'; absolute path: '/home/user/root_dir'; path with
                        variables: '$HOME/root_dir'.

ERROR: Directory /mnt/yarn/usercache/hadoop/appcache/application_1613996222010_0005/container_1613996222010_0005_01_000001/hdfs:/mnt/integration_tests' not found. Check your '--rootdir' option.